blob.py

```diff
@@ -132,9 +143,11 @@ def create(context, backend, input, output, format, theme, tempo, language):
         music21.converter.parse(input),
         theme=Theme[theme],
         language=language,
-        tempo=tempo
+        tempo=tempo,
+        tracks=tracks,
```

Comment:
Would need a bit of modular arithmetic or safeguards to avoid triggering the validation code with single-part scores. Maybe something like this:

```suggestion
        tracks=(0, 0, 0, 0) if len(data.stream.parts) == 1 else tracks,
```